test/evergreen/tcmalloc_install_or_build.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Make a tcmalloc shared library avilable for preloading into the environment.
+if [[ $# -ne 1 ]]; then
+    echo "Usage: $0 build-variant"
+    exit 1
+fi
+
```

Comment:
nit: For added safety, I'd recomment to use `set -u` to catch uses of any onset variables. This helps to avoid errors like `rm -rf $path1/$path2` with both variables unset.

Reply:
added